### DISCUSSION TITLE
Can't view profile after watching a replay

### DISCUSSION THREAD

**Discussion by Semro** [ppy/osu#17178] [Q&A] [answered]
On score panel, profile and it's avatar are not loading after watching a replay.
Similar issue: https://github.com/ppy/osu/issues/12864

osu!lazer 2022.306.0

https://user-images.githubusercontent.com/3332159/157301048-12d1219f-9d03-4d5c-906f-73dbb9913b1e.mp4

[runtime.log](https://github.com/ppy/osu/files/8208703/runtime.log)
[performance.log](https://github.com/ppy/osu/files/8208706/performance.log)
[network.log](https://github.com/ppy/osu/files/8208707/network.log)

**Comment by bdach** [comment]
see https://github.com/ppy/osu/issues/5966#issuecomment-527688172

**Reply by Semro** [reply]
Shouldn't that comment be expanded to a separate issue then? Because chronologically it seems that issue was resolved by #15487 and all referred issues too.

**Comment by smoogipoo** [comment, marked as answer]
https://github.com/ppy/osu/pull/17185
